plugins/dfu/dfu-device.c

```diff
@@ -1227,6 +1228,51 @@ dfu_device_open (FuUsbDevice *device, GError **error)
 		priv->status = DFU_STATUS_OK;
 	}
 
+	/* GD32VF103 encodes the serial number in UTF-8 (rather than UTF-16)
+	 * and also uses the first two bytes as the model identifier */
+	if (fu_device_has_custom_flag (FU_DEVICE (device), "gd32")) {
+#if G_USB_CHECK_VERSION(0,3,6)
```

Comment:
probably want to set subprojects to 0.3.6

Reply:
you okay with a git commit? 0.3.6 isn't out yet.

Reply:
oh hmm, probably better to wait until 0.3.6 then.  don't want to other unstable /untested stuff to push out from subprojecting.